rfc9000.md

```diff
@@ -6916,10 +6915,10 @@ that are received on a different network path.
 The Source and Destination Connection ID fields are the primary means of
 protection against an off-path attack during the handshake; see
 {{validate-handshake}}.  These are required to match those set by a peer.
-Except for Initial and Stateless Reset packets, an endpoint only accepts
-packets that include a Destination Connection ID field that matches a value the
-endpoint previously chose.  This is the only protection offered for Version
-Negotiation packets.
+Except for Initial and Stateless Resets, an endpoint only accepts packets that
```

Comment:
I'm not convinced that this is more consistent (here and all the Stateless Reset changes above). Just at the bottom of this paragraph, we say Version Negotiation packets. Given that the capital case represents packet types, what is the argument for removing packets? I'll admit it's redundant, but we use it quite a bit elsewhere, and I believe that it'd be more consistent to use `Stateless Reset packets`.

Reply:
Ah, but Version Negotiation is a packet (it is defined thus).  Stateless Reset is defined as a UDP datagram, not a packet.  That's why (at least in my opinion) the change is necessary.